buildlib/travis-build

```diff
@@ -20,6 +20,16 @@ cd ../build32
 CC=gcc-6 CFLAGS="-Werror -m32" cmake -GNinja .. -DENABLE_RESOLVE_NEIGH=0
 ninja
 
+# aarch64 build to check compilation on ARM 64bit platform
+cd ../build-aarch64
+if ! (which aarch64-linux-gnu-gcc &> /dev/null); then
+   AARCH64_PATH=`find / -type d -name 'gcc-linaro-*-x86_64_aarch64-linux-gnu' -print -quit 2>/dev/null | xargs echo`
+   export PATH=$PATH:${AARCH64_PATH}/bin
```

Comment:
Ah, right, but there is no reason for that, it was just copy and pasted from the rpm/deb flow where it does make sense.

Add this hunk, and setup HOME=/home/travis before doing the RUN of the before_script and it will more closely match the real travis.org environment.

```
diff --git a/buildlib/cbuild b/buildlib/cbuild
index 6ccf07a4a722f5..6bb7353401a3e9 100755
--- a/buildlib/cbuild
+++ b/buildlib/cbuild
@@ -534,7 +534,7 @@ def run_travis_build(args,env):
                                    "--git-dir",os.path.join(opwd,".git"),
                                    "reset","--hard","HEAD"]);
 
-        home = os.path.join(os.path.sep,"home",os.getenv("LOGNAME"));
+        home = os.path.join(os.path.sep,"home","travis");
 
         opts = [
             "run",
```